src/job_cache/schema.sql

```diff
@@ -95,5 +95,21 @@ create table if not exists output_symlinks(
   job            job_id  not null references jobs(job_id) on delete cascade);
 create index if not exists output_symlink_by_job on output_symlinks(job);
 
+-- We need to track the total size to know when to start collecting stuff
+create table if not exists total_size(
+  total_size_id integer primary key,
+  size          integer not null);
+
+insert into total_size (total_size_id, size) select 0, 0 where not exists (select * from total_size);
```

Comment:
Not sure if it matters, but an id of 0 _feels_ wrong

Reply:
If you use auto-increment it starts at 1 not 0 so you might have a point. It's not wrong in any precise way that I'm aware of however. I'll change it to 1.